go/chat/inboxsource.go

```diff
@@ -720,7 +720,9 @@ func (h convSearchHit) hitScore() (score int) {
 			score += 3
 		}
 	}
-	if len(h.queryToks) == len(h.convToks) && len(h.hits) == len(h.convToks) {
+	if len(h.queryToks) == len(h.convToks) &&
+	len(h.hits) == len(h.convToks) &&
+	exacts == len(h.hits)  {
 		if exacts == len(h.hits) {
```

Comment:
heh you should remove this too though.. sorry the suggestion interface isn't great for the multi line changes

Reply:
ah sorry